app/authorities/sufia/authorities/local/find_works.rb

```diff
@@ -0,0 +1,21 @@
+module Sufia::Authorities
+  class Local::FindWorks < Qa::Authorities::Base
+    attr_reader :q
```

Comment:
@andjsmit Okay, I looked more closely. Do not call `register_subauthority` and do not define an `initialize` method. This should just work because it is not a subauthority. It is equivalent to the built in tgnlang authority: https://github.com/projecthydra-labs/questioning_authority/blob/master/lib/qa/authorities/tgnlang.rb